src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/responses/Message.kt

```diff
@@ -35,6 +36,8 @@ data class Message(
     val sessionId: String,
     val messageType: String,
 
+    val attachedEventIds: Set<String>?,
```

Comment:
Why can we use not-null type for `attachedEventIds`? I think if the `attachedEventIds` collection is empty it will be enough to tell that we don't need (or don't have) attached events